libiscsi/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu
```

Comment:
Can this be checked if it runs with alpine? Just to reduce the size of the image, in case it works.

Reply:
libiscsi is not built for alpine and the effort to get it running (install all dependencies during compile)  is considerable. Since this is a compliance suite - the expectation is that it is a one-time run should any "user" attempt to use it. 

Having said that, agree on the intent. Have replaced ubuntu w/ debian which I found to be next best wrt size and also having support for the libiscsi package. It is ~35 MB lesser.

The tests were successfully run w/ the updated dockerfile.